src/main/java/org/folio/innreach/domain/service/impl/CirculationServiceImpl.java

```diff
@@ -429,6 +433,7 @@ private void initiateTransactionHold(String trackingId, String centralCode,
                                        TransactionType transactionType,
                                        BiConsumer<InnReachTransaction, Boolean> postProcessor) {
 
+    transactionHold.setPatronName(normalizePatronName(transactionHold.getPatronName()));
```

Comment:
this method is not suitable for such logic, since it's used for all types of transactions while patron name needs to be manually added only for PATRON hold